Rewrite bot in TS

TechGeekGamer opened the pull request.

hax4dazy commented:
I am hax4dayz and I approve of this pull request

Slluxx reviewed (commented):
see included comments

hax4dazy commented on src/commands/support.ts:
```diff
@@ -0,0 +1,86 @@
+            .setTitle("Guiformat")
+            .setURL('http://ridgecrop.co.uk/guiformat.exe')
+            .setDescription('A useful tool for formatting SD cards over 32GB as FAT32 on Windows.')
+            message.channel.send("The exFAT drivers built into the Switch has been known to corrupt SD cards and homebrew only makes this worse. Backup everything on your SD card as soon as possible and format it to FAT32. On Windows, if your SD card is over 32GB then it will not let you select FAT32 from the built-in format tool, however you can use a tool like GUIFormat to format it.", {embed:e ...
```
Can we change this text to the same text that behemoth has. We'd have to reword it a bit tho to be more user friendly

Slluxx replied:
how about a suggestion @hax4dazy :harold:

hax4dazy replied:
Oh I rewrote the text and DM'ed it to him. Let me put it here tho.

`The standard exFAT driver is bad and shouldn't be used.

If your PC doesn't allow you to format your card to FAT32 you can do that in hekate
Tools -> Arch bit · RCM · Touch · Partition -> Partition`

Slluxx replied:
how about "should be avoided at all cost" ?
and isnt "arch bit" actually "archive bit" ? idk though, never used it.

hax4dazy replied:
Nope its actually called arch bit. Also yeah "should be avoided at all cost" does sounds better.

TechGeekGamer replied:
> Can we change this text to the same text that behemoth has. We'd have to reword it a bit tho to be more user friendly

Updated in e964cde69e793807ca3e3c1b6bb7b83a4c3a3fe2

TechGeekGamer commented on src/commands/support.ts:
```diff
@@ -0,0 +1,86 @@
+import { MessageEmbed } from 'discord.js'
+import {config} from '../../config'
+export const supportCommands:Array<Command> = [
+    {
```
Added in e964cde69e793807ca3e3c1b6bb7b83a4c3a3fe2

hax4dazy reviewed (changes requested):
I changed the text a bit in [this](https://github.com/Team-Neptune/Korral-JS/pull/11#discussion_r635475302) discussion

TechGeekGamer commented:
> I changed the text a bit in [this](https://github.com/Team-Neptune/Korral-JS/pull/11#discussion_r635475302) discussion

b50e92347b0f906241fb863031920b984a654054

TechGeekGamer commented on src/commands/moderation.ts:
```diff
@@ -0,0 +1,330 @@
+    
+            writeFileSync('./warnings.json', JSON.stringify(warnings))
+            var eventMessage = `You were warned on ${message.guild.name}.\nThe given reason is: ${reason}\n\nPlease read the rules. This is warning #${(warnings[mentionedUser.id].length)}.`
+            switch (warnings[mentionedUser.id].length) {
```
How would the `message` be used?

Slluxx replied:
its the `eventMessage` text

TechGeekGamer replied:
Alright how does the setup in d84f5ebae3dce295972cc016a36b182e7dc9e5c3 look?

Slluxx replied:
looks good to me

hax4dazy commented:
> 
> 
> > I changed the text a bit in [this](https://github.com/Team-Neptune/Korral-JS/pull/11#discussion_r635475302) discussion
> 
> [b50e923](https://github.com/Team-Neptune/Korral-JS/commit/b50e92347b0f906241fb863031920b984a654054)

You still missed some things. Please copy the message and paste it in there

Slluxx reviewed (approved):
merge and see what breaks i guess

TechGeekGamer commented:
> > > I changed the text a bit in [this](https://github.com/Team-Neptune/Korral-JS/pull/11#discussion_r635475302) discussion
> > 
> > 
> > [b50e923](https://github.com/Team-Neptune/Korral-JS/commit/b50e92347b0f906241fb863031920b984a654054)
> 
> You still missed some things. Please copy the message and paste it in there

Updated it in e1abe1cba7772326d31c32d4d9a8d6ef7a098bd6, how does it look?
